packages/story-editor/src/output/page.js

```diff
@@ -41,6 +42,7 @@ function OutputPage({ page, autoAdvance = true, defaultPageDuration = 7 }) {
     elements,
     backgroundColor,
     backgroundAudio,
+    tracks = [],
```

Comment:
Maybe `tracks: backgroundAudioTracks = []` for improved readability?

So further down it's clearer that we deal with `videoTracks` and `backgroundAudioTracks`

Reply:
Fixed https://github.com/GoogleForCreators/web-stories-wp/pull/10288/commits/c4702cdff4a2b35dc06052d7e3d0c6898bd4696d